.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: ci
+on: [push]
```

```suggestion
on: [push, pull_request]
```

It would be good to see that PRs are passing, I think this would make it work.

---

Oh.. actually now I remember why I didn't add it. PR builds also run in the context of the source repo, but don't contain the secrets (e.g. `NPM_TOKEN`), for obvious security reasons. Also we shouldn't release with every new PR build (`yarn dv-scripts ci`).

---

I addressed this by changing the conditional to:

```
if: github.event.repository.fork == false && github.event_name != 'pull_request'
```

---

I think I have the rethink how `dv-scripts` works a bit to simplify this 😆 . Good for now though!